src/Forms/ResourceLocatorField.php

```diff
@@ -80,6 +80,15 @@ public function getSchemaDataDefaults()
         ]);
     }
 
+    public function performReadonlyTransformation()
+    {
+        // Clear out the description that's only relevant when the field is editable.
+        $this->setDescription('');
```

Comment:
This should be done on the cloned instance rather than the current

Reply:
Done